src/ui/components/shared/Button.tsx

```diff
@@ -69,7 +69,10 @@ function getColorClasses(color: Colors, style: ButtonStyles) {
 export function getButtonClasses(color: Colors, style: ButtonStyles, size: ButtonSizes) {
   const standardClasses = STANDARD_CLASSES[size];
   const colorClasses = getColorClasses(color, style);
-  const focusClasses = "focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500";
+  const focusClasses = `focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-${getColorCode(
+    color,
```

Comment:
Changed the color of the focus ring to match the button's color per Jon's request.

![Screen Shot 2021-11-11 at 1 21 26 PM](https://user-images.githubusercontent.com/478109/141349706-3fa77fe0-ac97-402c-81ca-225b7d846dfb.png)